pkg/ocm/output/machinepools.go

```diff
@@ -40,6 +40,20 @@ func PrintLabels(labels map[string]string) string {
 	return strings.Join(output, ", ")
 }
 
+func PrintUserAwsTags(tags map[string]string) string {
+	if len(tags) == 0 {
+		return ""
+	}
+	output := []string{}
+	for k, v := range tags {
+		if !strings.HasPrefix(k, "aws:") {
```

Comment:
Could it have aws prefix if we do not allow it to be supplied? Considering we are not fetching it from AWS I'd say this wouldn't ever show up :thinking:

Reply:
yeah you're right, just removed